schema/ingestion_config/v1.0.0/ingestion_config_models.yaml

```diff
@@ -54,6 +54,10 @@ classes:
         description: Annotations for the dataset.
         multivalued: true
         range: AnnotationEntity
+      collection_metadata:
```

Comment:
Extended validation for this should be added as well - I can hop on a call and show you how to do this, if you'd like. Thanks!